src/utils/getValidJson.ts

```diff
@@ -2,7 +2,7 @@
  * Checks if string is in valid JSON format else throw error.
  * @param str - string to check.
  */
-export const isValidJson = (str: string | object) => {
+export const getValidJson = (str: string | object) => {
```

Comment:
I would prefer to have two separate utils.

Reply:
We are parsing JSON to check if it's valid so since we don't want to parse again and again we decided to return it after checking to optimize the process.